tests/Doctrine/Tests/DBAL/Platforms/OraclePlatformTest.php

```diff
@@ -697,6 +697,60 @@ public function testQuotedTableNames()
     }
 
     /**
+     * @dataProvider getReturnsGetListTableColumnsSQL
+     * @group DBAL-831
+     */
+    public function testReturnsGetListTableColumnsSQL($database, $expectedSql)
+    {
+        $this->assertEquals($expectedSql, $this->_platform->getListTableColumnsSQL('"test"', $database));
+    }
+
+    public function getReturnsGetListTableColumnsSQL()
```

Comment:
We tend to avoid testing against complete SQL strings when it comes to testing `getList*()` Methods as those tests are fragile and need to be adjusted on every change in SQL. I would rather do a contains check in the string to check whether `all_tab_columns` or `user_tab_columns` is present (which is the interesting difference).